xbmc/music/karaoke/GUIDialogKaraokeSongSelector.h

```diff
@@ -83,5 +83,5 @@ class CGUIDialogKaraokeSongSelectorLarge : public CGUIDialogKaraokeSongSelector
 {
   public:
     CGUIDialogKaraokeSongSelectorLarge();
-    void Open();
+    void Open(const std::string &param = "");
```

Comment:
This one isn't an override by the looks (Open in base isn't virtual) so the only way this one does anything is if it's getting called on the class directly (which, given the karaoke code, is almost certain).

It doesn't hurt I guess, but also could cause confusion with folk thinking it's an override when it's not (and shouldn't be in my view). I'd track down why this is even there. By the looks something to do with the fact that the Small version takes an integer into it's (also non-overriding) version of Open().